pages/entropy/error-codes.mdx

```diff
@@ -0,0 +1,21 @@
+# Error Codes
+
+The following table contains the errors used in the Pyth Network's Entropy [EVM contracts](https://github.com/pyth-network/pyth-crosschain/blob/d290f4ec47a73636cf77711f5f68c3455bb8a8ca/target_chains/ethereum/contracts/contracts/entropy/Entropy.sol).
+This information is derived from [EntropyErrors.sol](https://github.com/pyth-network/pyth-crosschain/blob/d290f4ec47a73636cf77711f5f68c3455bb8a8ca/target_chains/ethereum/entropy_sdk/solidity/EntropyErrors.sol)
+in the Pyth EntropySDK and can be used to decode error codes programmatically.
+
+| Error Codes | Error                       | Error Description                                                                                                                                                                                                           |
+| ----------- | --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0xd82dd966  | AssertionFailure()          | Contract invariant failed.                                                                                                                                                                                                  |
+| 0xda041bdf  | ProviderAlreadyRegistered() | Provider already registered.                                                                                                                                                                                                |
+| 0xdf51c431  | NoSuchProvider()            | Requested Provider does not exist.                                                                                                                                                                                          |
+| 0xc4237352  | NoSuchRequest()             | Request does not exist.                                                                                                                                                                                                     |
+| 0x3e515085  | OutOfRandomness()           | Provider is our of committed random numbers.                                                                                                                                                                                |
+| 0x025dbdd4  | InsufficientFee()           | Transaction fee insufficient.                                                                                                                                                                                               |
```

Comment:
Transaction fee -> Request fee.
Use getFee function to get the latest fee necessary for a request.